src/components/lco/PageBanner.tsx

```diff
@@ -36,62 +44,101 @@ type PageBannerProps = Pick<ClimbType, 'ancestors'>
  */
 export const PageBanner: React.FC<PageBannerProps> = ({ ancestors }) => {
   const orgs = findLCOs(LCO_LIST, ancestors)
-  if (orgs.length === 0) return null
   return (
-    <div className='mt-8 bg-area-cue/20 p-6 rounded-box'>
-      <div className='mt-2 mb-1 text-xs text-base-300'>Local climbing organizations</div>
-      <div className='flex flex-col'>
-        {orgs.map(orgProfile => <IndividualBanner key={orgProfile.id} profile={orgProfile} />)}
+    <div className='grid pt-6 pb-4 lg:pb-16 lg:pt-16'>
+      <div className='col-span-full flex justify-start items-center pb-6'>
+        <h3 className='mr-4'>Local climbing organizations</h3>
+        <Tooltip content={
+          <p>Learn more about our&nbsp;
+            <a href='https://openbeta.substack.com/p/openbeta-and-lcos' target='_blank' rel='noreferrer' className='underline'>
+              initiative
+            </a>.
+          </p>
+        }
+        >
+          <InformationCircleIcon className='h-6 w-6' />
+        </Tooltip>
+      </div>
+      <div>
+        {orgs.length === 0
+          ? (
+            <p className='italic text-base-content/60'>
+              No organizationa found for this area
+            </p>
+            )
+          : (
+              orgs.map((orgProfile) => (
+                <IndividualBanner key={orgProfile.id} profile={orgProfile} />
+              ))
+            )}
       </div>
     </div>
   )
 }
 
 const IndividualBanner: React.FC<ContentProps> = ({ profile }) => (
-  <HoverCard.Root openDelay={300}>
-    <HoverCard.Trigger>
-      <ContentTrigger profile={profile} />
-    </HoverCard.Trigger>
-    <HoverCard.Portal>
-      <HoverCard.Content sideOffset={10} side='top' className='z-40'>
-        <Card profile={profile} />
-      </HoverCard.Content>
-    </HoverCard.Portal>
-  </HoverCard.Root>
+  <>
+    <div className='sm:inline-block mr-6 mb-6'>
```

Comment:
what is the intention here? `sm:inline-block`

I understand this as for  anything smaller `sm` don't make it an inline-block (default = block).  Anything sm, md, lg and larger -\> inline-block

https://tailwindcss.com/docs/responsive-design#working-mobile-first

Reply:
Good catch.
In my experiment,  for the web layout to work, each div needs to be `inline-block` when using grid layout( in my understanding at least). On mobile, this is not required. So my intention was to only change the display to `inline-block` on web(`sm` breakpoints and onwards). I wasn't targeting mobile with `sm:inline-block`